java/oauth-custom-challengehandler/README.md

```diff
@@ -0,0 +1,31 @@
+# Authenticate with OAuth
+
+This sample demonstrates how to authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires a login on the portal that hosts them (an ArcGIS Online account, for example).
+
```

Comment:
I noticed that the description doesn't mention that this sample uses a WebView to show the OAuth sign-in page. I think that's quite important to highlight.

Reply:
updated